### PR TITLE
Add HTTP tests for Web & Connect APIs

### PR DESCRIPTION
**Pull request by luchaos**
... to quickly test for PHP 8 compatibility.
I think those are for JetBrains IDEs only, at least I could not find any reference as to whether those can be run by another HTTP client. Added curl notations for reference.

Includes all of Web API but only a few Connect API requests as the latter was acceptance tested with an actual client (latest RALibretro).

### CODE REVIEW

File: tests/http/http-client.env.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "dev": {
+    // Override these in http-client.private.env.json
```
Comment (luchaos) [inline tests/http/http-client.env.json:3]: PHPStorm HTTP client ignores invalid comments in JSON files

File: public/API/API_GetAchievementOfTheWeek.php [hunk 2]
```diff
@@ -15,7 +15,7 @@
         'Achievement' => ['ID' => null],
         'StartAt' => null,
     ]);
-    return;
+    exit;
```
Comment (luchaos) [inline public/API/API_GetAchievementOfTheWeek.php:18]: Prevent any further execution

**Review by Jamiras — APPROVED** [review]
If you believe these are JetBrains specific, you should document as such with a comment at the top of the files. I certainly don't recognize the format for any tools I've used.

**Comment by luchaos** [comment]
> If you believe these are JetBrains specific, you should document as such with a comment at the top of the files. I certainly don't recognize the format for any tools I've used.

Good idea - done 👍

**Pull request merged**